tests/transforms/test_control.py

```diff
@@ -0,0 +1,192 @@
+from functools import partial
+import numpy as np
+import pennylane as qml
+from pennylane.tape import QuantumTape
+from pennylane.transforms.control import ctrl, ControlledOperation
+from pennylane.tape.tape import expand_tape
+
+
+def assert_equal_operations(ops1, ops2):
+    assert len(ops1) == len(ops2)
+    for op1, op2 in zip(ops1, ops2):
+        assert type(op1) == type(op2)
+        assert op1.wires == op2.wires
+        np.testing.assert_allclose(op1.parameters, op2.parameters)
+
+
+def test_control_sanity_check():

```

Comment:
Don't forget the docstrings for all tests. I know they are annoying to write, but it helps tremendously to get more information on the test logic.

Reply:
Agree, especially in this case where sanity check is not well defined!

Reply:
Added